### ISSUE TITLE
Is there any way to load symlinks running rspec-command?

### ISSUE THREAD

**Issue by alexvbush** [coderanger/rspec-command#2]
Hi, I'm trying to use rspec-command as a higher level integration test for non-ruby command line app. I'd like to define my specs using rspec and have it run say a swift or python command line app when the specs are executed. It seems like rspec-command isn't aware of symlinks in the directory where I run it. Is there a way to have those symlinks be linked?

**Comment by coderanger** [comment]
What kind of errors are you seeing? Symlinks are just kind of a thing that exists, you don't load them or not.

**Comment by alexvbush** [comment]
let me rephrase because I realize I wasn't very clear as I reread what I wrote above :)

I have a command line app written in swift. I would like to use rspec-command to integration test it.

The idea is that I will define high level integration tests and behavior of my command line app using rspec and then later I can swap implementation using completely different language - ruby, swift, python, whatever.

Currently I have a ruby bundler project set up that has rspec-command added to it. 
I also have my swift project as a subfolder of this ruby project. And I have the output binary of swift code synlinked from that swift subfolder up to the root of ruby project where Gemfile and spec folders live.

When I run `command 'time_calculator 12/24/18 add 90 days'` I'm getting the following error even though running the same command from terminal manually actually works:

```
Failures:

  1) Time Calculator High Level Behavior commands time addition days should add given number of days to a given date stdout
     Failure/Error: its(:stdout) { should eq "Sun, 24 Mar 2019\n" }

     Mixlib::ShellOut::ShellCommandFailed:
       Expected process to exit with [0], but received '127'
       ---- Begin output of bundle exec time_calculator 12/24/18 add 90 days ----
       STDOUT:
       STDERR: bundler: command not found: time_calculator_backup
       Install missing gem executables with `bundle install`
       ---- End output of bundle exec time_calculator 12/24/18 add 90 days ----
       Ran bundle exec time_calculator_backup 12/24/18 add 90 days returned 127
     # ./spec/time_calculator_integration_spec.rb:26:in `block (6 levels) in <top (required)>'

Finished in 0.25723 seconds (files took 0.11814 seconds to load)
1 example, 1 failure

Failed examples:

rspec ./spec/time_calculator_integration_spec.rb:26 # Time Calculator High Level Behavior commands time addition days should add given number of days to a given date stdout

Randomized with seed 60167

```

**Comment by alexvbush** [comment]
Basically I want to figure out how to make my other executable be available in rspec environment that `rspec-command` runs in.

maybe I'm going about it entirely the wrong way but the idea is kind of like capybara - you define high level integration tests and behavior of a website in ruby/rspec and then execute it/run tests. Your actual underlining implementation could be written in anything because capybara communicates with it through language agnostic http/html interface.

In my case I want to have integration tests that use command line interface as the language agnostic communication tool. Rspec is so nice to write/define tests/behavior but I don't necessarily want to implement my command line in ruby though.

**Comment by coderanger** [comment]
So every test is run in a temp folder for isolation reasons. You would probably want to use https://github.com/coderanger/rspec-command#fixture_file to copy over your command into the temp dir :)

**Comment by alexvbush** [comment]
ha, it worked, thanks!

one thing though, now I have to have `./` before the name of my command:

```ruby
describe 'swift days' do
  fixture_file 'swift_time_calculator'
  command './swift_time_calculator 12/24/18 add 90 days'
  its(:stdout) { should eq "Sun, 24 Mar 2019\n" }
end
```

any way to get rid of it?

Also, one more little thing - I tried multiple things but couldn't figure out how to assets without `its`. The docs say that the subject is `Mixlib::ShellOut.new` but it seems like I'm getting a `String` instead.

**Comment by coderanger** [comment]
So the subject in RSpec-speak is the thing you get when you use `it`. `its(:stdout)` means you are testing against `subject.stdout`, which works because `subject` is a ShellOut object. As for the `./`, probably the easiest option is to make a little helper method:

```ruby
def my_command(args)
  fixture_file 'swift_time_calculator'
  command '/swift_time_calculator #{args}"
end
```

Or something like that :)

**Comment by alexvbush** [comment]
makes sense, thank you for your help! will try tonight/this weekend.
